posts/h5ad_conversion/index.qmd

```diff
@@ -0,0 +1,621 @@
+conda create --name myenv python==3.9.18 pandas==2.1.0 anndata==0.10.2
+```
+
+To activate an existing environment, one should pass the name of environment to `conda activate` function.  
+```{default filename="Terminal"}
+conda activate myenv
+```
+:::
+
+To launch the viewer for the `.h5ad` object of interest, one can simply run
+```{default filename="Terminal"}
+cirro launch <path_to_dataset_file.h5ad>
+```
+
+The cirrocumulus viewer allows user to 
+
+* view multiple cell embeddings (`umap` and `spatial`) side-by-side for both cell metadata and gene expression (@fig-full-embedding, Dual Embedding View); 
+
+* use lasso tool to subset cells of interest in the embedding space (@fig-chosen, Lasso-in-Space);
+
+* explore how the gene expression changes between different selections (@fig-marker-heatmap, Marker Heatmap for all cells vs. @fig-dot-plots, Dot Plots for Subsets). 
+
+For more information, please refer to the [documentation](https://cirrocumulus.readthedocs.io/en/latest/documentation.html){target="_blank"} of `cirrocumulus` package
+
+::: {.panel-tabset}
+
+## Dual Embedding View
+```{r}
+#| eval: true
+#| echo: false
+#| label: "fig-full-embedding"
+#| fig-cap: "The dual-embedding views of `umap` and `spatial` for chosen cell meta data and gene expression level."
+knitr::include_graphics("./figures/Fig-mBrain-S1-cirroViewer-screenshot.png")
+```
+
+## Lasso-in-Space
+```{r}
+#| eval: true
+#| echo: false
+#| label: "fig-chosen"
+#| fig-cap: "Use lasso tool to select cells in `spatial` embedding. Chosen cells are shown in solid colors."
+knitr::include_graphics("./figures/Fig-mBrain-S1-lasso.png")
+```
+
+## Marker Heatmap
+```{r}
+#| eval: true
+#| echo: false
+#| label: "fig-marker-heatmap"
+#| fig-cap: "Heatmap of selected gene expression across 'leiden_clus' of all cells."
+knitr::include_graphics("./figures/Fig-mBrain-S1-cirroViewer-markerHeatmap.png")
+```
+
+## Dot Plots for Subsets
+```{r}
+#| eval: true
+#| echo: false
+#| label: "fig-dot-plots"
+#| fig-cap: "Dot plots of chosen gene expression for all cells (top) vs. cells within lasso selection (bottom) in @fig-chosen tab."
+knitr::include_graphics("./figures/Fig-mBrain-S1-cirroViewer-dotPlot-selection-vs-all.png")
+```
+
+:::
+
+
+# Creating per-sample `anndata` object from basic data files in `Python` for python-based single-cell analysis {#sec-create-from-flatFiles}
+
+For python lovers, it may be desired to create `anndata` object from scratch and then feed it into python-based single-cell analysis tools, like [`scanpy`](https://scanpy.readthedocs.io/en/stable/){target="_blank"} and [`squidpy`](https://squidpy.readthedocs.io/en/stable/index.html){target="_blank"}.
+
+In this section, we start from the basic data files (in `.csv` format) exported from AtoMx&#8482; SIP for an example public dataset that you can download from [here](https://nanostring.com/products/cosmx-spatial-molecular-imager/ffpe-dataset/cosmx-smi-human-pancreas-ffpe-dataset/){target="_blank"}. 
```

Comment:
typo: `In this section, we start from the basic data files (in `.csv` format) exported from AtoMx&#8482; SIP. For an example public dataset that you can download, go [here](https://nanostring.com/products/cosmx-spatial-molecular-imager/ffpe-dataset/cosmx-smi-human-pancreas-ffpe-dataset/){target="_blank"}.`